app/dashboard/templates/profiles/tab_follow.html

```diff
@@ -0,0 +1,145 @@
+          </div>
+        </div>
+        <div class="mb-0 mt-1 col-5 d-flex flex-row align-items-center">
+        <div class="mr-5" style="height: fit-content">
+          <button class="btn btn-outline-gc-blue btn-sm flex-grow-1 font-smaller-5 position-relative quick-link" data-openchat="{{user.org}}">
+            <i class="fas fa-comment-dots" data-p lacement="bottom" data-toggle="tooltip" data-html="true"
+              title="Chat @{{ user.org.handle }}">
+            </i>
+          </button>
+        </div>
+        <div style="height: fit-content">
+          <a class="btn btn-sm flex-grow-1 mr-5 font-smaller-5 position-relative quick-link btn-outline-gc-blue" {% if not user.org.email %} hidden {% endif %} href="mailto:{{user.org.email}}"  data-placement="bottom" data-toggle="tooltip" data-html="true" title="@{{ user.org.handle }}'s Email">
+            <i class="far fa-envelope"></i>
+          </a>
+        </div>
+        <div class="d-flex align-items-center" style="height: fit-content">
+          {%  with followers=user.mutual_follower %}
+            {% if followers|length == 1 %}
+                <div style="width: 60px">
+                    <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ followers.0.profile.handle }}" alt="">
+                </div>
+                <p class="text-muted font-smaller-4 my-0">Followed by {{ followers.0.profile.handle }}</p>
+            {% endif %}
+            {% if followers|length == 2 %}
+                <div style="width: 60px" class="d-flex flex-row align-items-center">
+                    <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px " src="/dynamic/avatar/{{ followers.0.profile.handle }}" alt="">
+                    <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ followers.1.profile.handle }}" alt="">
+                </div>
+                <p class="text-muted font-smaller-4 my-0">Followed by {{ followers.0.profile.handle  }} and {{ followers.1.profile.handle }}</p>
+            {% endif %}
+            {% if followers|length >= 3 %}
+                <div style="width: 60px" class="d-flex flex-row align-items-center">
+                    <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px" src="/dynamic/avatar/{{ followers.0.profile.handle }}" alt="">
+                    <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px" src="/dynamic/avatar/{{ followers.1.profile.handle }}" alt="">
+                    <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ followers.2.profile.handle }}" alt="">
+                </div>
+                <p class="text-muted font-smaller-4 my-0">Followed by {{ followers.0.profile.handle  }}, {{ followers.1.profile.handle }} and {{ followers|length|add:-2}} others you follow</p>
+            {% endif %}
+          {% endwith %}
```

Comment:
lets move this logic to a loop instead and limit the looping to be more DRY

Reply:
I'm not sure this would be better as a loop

Reply:
done